packages/modules/cra-template-itwin-viewer/template/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```

Comment:
maybe add some steps pertaining to the viewer, like adding the project & imodel id in the .env file, etc

Reply:
done